package.json

```diff
@@ -35,7 +35,8 @@
     "prettier": "./prettier-commits.sh",
     "prettier-on": "prettier --write",
     "test": "jest",
-    "db:setup": "psql \"$DATABASE_URL\" -c 'CREATE EXTENSION IF NOT EXISTS \"uuid-ossp\";'",
+    "db:setup": "psql \"$DATABASE_URL?sslmode=prefer\" -c 'CREATE EXTENSION IF NOT EXISTS \"uuid-ossp\";'",
+    "db:setup:heroku": "psql \"$DATABASE_URL?sslmode=prefer\" -c 'CREATE EXTENSION IF NOT EXISTS \"uuid-ossp\" WITH SCHEMA heroku_ext;'",
```

Comment:
I had to split these two as heroku has it's on schema right now.
It seems like there will be a way to use the public schema, but I wanted to make it work without enabling addons

https://help.heroku.com/ZOFBHJCJ/heroku-postgres-extension-changes-faq